### PR TITLE
Overwrite bundled libgfortran with system libgfortran if it is newer …

### PR DESCRIPTION
**Pull request by mvdbeek**
…as in PR #199.

Unfortunately I couldn't reproduce @bcclaywell's issue #325 on a clean ubuntu 14.04 install,
so it is untested right now.

### CODE REVIEW

**Comment by mvdbeek** [comment]
@bgruening can I upload this to the TTS before merging? Hoping for some feedback

File: packages/package_r_3_2_1/tool_dependencies.xml [hunk 1]
```diff
@@ -9,6 +9,20 @@
                         <source_directory>.</source_directory>
                         <destination_directory>$INSTALL_DIR</destination_directory>
                     </action>
+                    <action type="shell_command"><![CDATA[
+                        command -v gfortran || return 0
```
Comment (yhoogstrate) [inline packages/package_r_3_2_1/tool_dependencies.xml]: Is this functional?

**Comment by bgruening** [comment]
@mvdbeek yes please put it on TTS and start testing intensively ...

**Comment by bgruening** [comment]
@bcclaywell could you please test this as well?
If you have an installation, please search for your tool_dependencies.xml, patch it and reinstall the dependency (not the package). This should be the easiest way to test it.

File: packages/package_r_3_2_1/tool_dependencies.xml (hunk 1 above)
Comment (mvdbeek) [inline packages/package_r_3_2_1/tool_dependencies.xml]: @yhoogstrate 
should be working. 
See https://github.com/galaxyproject/tools-iuc/pull/199#issuecomment-117710086
If I understand well the idea is that you avoid overwriting the shipped libgfortran if no fortran is installed on the system.

**Comment by mvdbeek** [comment]
updated the TTS, i forgot about this PR.
Haven't encountered additional problem during testing, but i didn't have a problem to begin with ...

**Comment by bcclaywell** [comment]
@bgruening @mvdbeek I'm still seeing the same errors as in #325 after patching R 3.2.1's `tool_dependencies.xml` and reinstalling the dependency as @bgruening suggested. It looks like `libgfortran.so.3.0.0` has been replaced but `libgfortran.so.3` is still the bundled version.

```
$ ls -l *fortran*
lrwxrwxrwx 1 bclaywel galaxy_adm      16 Nov  9 16:34 libgfortran.so -> libgfortran.so.3
-rw-r--r-- 1 bclaywel galaxy_adm  964408 Aug 24 09:20 libgfortran.so.3
-rw-r--r-- 1 bclaywel galaxy_adm 1153392 May 14 11:48 libgfortran.so.3.0.0
-rw-r--r-- 1 bclaywel galaxy_adm  964408 Aug 24 09:20 libgfortran.so.3.0.0.bundled
```

**Comment by mvdbeek** [comment]
@bcclaywell Okay, thanks for the feedback. So after reading up on the issue (http://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html) libgfortran.so.3 ("Fully qualified soname") should be a symbolic link to the "realname" libgfortran.so.3.0.0 and a symbolic link libgfortran.so ("linker name") should point to libgfortran.so.3. I'll make the changes and let you know.

**Comment by mvdbeek** [comment]
@bcclaywell @bgruening This seems to be working for me now.
It's uploaded on the TTS.

**Comment by bgruening** [comment]
LGTM. @bcclaywell any chance you can test this again?

**Comment by bcclaywell** [comment]
That did the trick. I'm no longer seeing any errors during the DESeq2 install. Thanks!

**Comment by bgruening** [comment]
@bcclaywell awesome, thanks for testing.
@mvdbeek do we need to port this fix to any other repositories?

**Pull request merged**